website/src/content/docs/docs/emitters/sdk-clients/introduction.md

```diff
@@ -0,0 +1,102 @@
+---
+title: introduction of SDK client emitters
+---
+
+## How to Use Client Emitters to Generate SDK from TypeSpec
+
+### Introduction
+
+This guide will walk you through the process of using different client emitters (JavaScript, Python, Java, .NET) to generate SDKs from TypeSpec. Please note that all client emitters are currently in **preview** and are subject to changes in future versions.
+
+By following this guide, you will learn:
+
+1. How to set up and configure client emitters.
+2. Update the client emitter configurations in `package.json` and `tspconfig.yaml`.
+3. How to run the SDK generation for each specific programming language.
+
+## Location of All Client Emitters and Common Configurations
+
+The client emitters and their common configurations are located in the `package.json` file within your project.
+
+| **Emitter Name**             | **Language** | **Version**              | **Common Configuration** |
+| ---------------------------- | ------------ | ------------------------ | ------------------------ |
+| @azure-tools/typespec-ts     | JavaScript   | `0.37.0`                 | `emitter-output-dir`     |
+| @typespec/http-client-python | Python       | `0.6.0`                  | `emitter-output-dir`     |
+| @typespec/http-client-java   | Java         | `0.1.6`                  | `emitter-output-dir`     |
```

Comment:
Could we use `latest` in package version?

I am not sure whether providing a specific version would be helpful for customer, while dev could forget to update the number here when releasing new version of the lib, and making this doc outdated.

Reply:
I added the below notes:
```
Note: Check for the Latest Version

To ensure you are using the latest version of the packages, visit npmjs.com and search for the relevant packages.
```

How do you think of the options of 
a. always use latest 
b. every time, there is a emitter upgrade, we update the version in this file